Amplify/Categories/API/ClientBehavior/AmplifyAPICategory+ReachabilityBehavior.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 import Combine
 extension AmplifyAPICategory: APICategoryReachabilityBehavior {
     @available(iOS 13.0, *)
-    public func reachability(apiName: String?) throws -> AnyPublisher<ReachabilityUpdate, Never>? {
+    public func reachability(apiName: String? = nil) throws -> AnyPublisher<ReachabilityUpdate, Never>? {
```

Comment:
"Defaulting paramter, but doesn't seem to work" as in, even though you this here, `Amplify.API.reachability()` complains about missing argument?